### DISCUSSION TITLE
Using Magick.NET library on Kubernetes (AKS)

### DISCUSSION THREAD

**Discussion by fkucuk** [dlemstra/Magick.NET#895] [Help]
Hi,

I've created an ASP.Net 3.1 Web Api project and used Magick.NET as a helper library to resize and compress some images.

The app works fine on my Windows computer. But when I deploy it on my AKS cluster, my service returns some weird characters instead of the resized image?

From the documentation, it says I should use Magic.NET-Q8-x64 nuget package. I did, but the problem continues. 

Any suggestions?

**Comment by dlemstra** [comment]
That sounds like a problem inside your service? And without any more details I cannot really help you.

**Reply by fkucuk** [reply]
Hi @dlemstra, great tool by the way :),

I just want to be sure that it is sufficient to use the following package in my project to make it work on kubernetes (Linux) environment. I am just trying to resize jpeg images, so I believe Ghost Script is not necessary.

`<PackageReference Include="Magick.NET-Q8-x64" Version="7.23.3" />`

I am not sure which details you wanted to know?

**Reply by dlemstra** [reply]
The reason this is happening is probably related to how you coded your service. And this library should work in AKS

**Reply by fkucuk** [reply]
Well it turns out; I was returning FileContentResult from my service. And I was testing with Postman on AKS environment, where with Google Chrome on my development environment. And, apparently, Postman did not like FileContentResult. 

The tool works fine :D 

Thanks,
